frontend_tests/node_tests/hotkey.js

```diff
@@ -402,4 +410,52 @@ function stubbing(func_name_to_stub, test_function) {
     assert_mapping('down_arrow', 'drafts.drafts_handle_events');
     overlays.is_active = return_false;
     overlays.drafts_open = return_false;
+
+    assert_unmapped('narrow_compose', false, false, true);
```

Comment:
We can greatly simplify the hotkey tests by just having a single function in narrow.js that does all the heavy lifting.  If you read the rest of the hotkey code, you'll see that most keys just dispatch to a single function in some other module that does all the heavy lifting.